modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/notifications/test/JournalUserNotificationTest.java

```diff
@@ -142,9 +196,17 @@ protected BaseModel<?> updateBaseModel(BaseModel<?> baseModel)
 			(JournalArticle)baseModel, true);
 	}
 
+	private void _activateSingleApproverWorkflow() throws Exception {
+		_workflowDefinitionLinkLocalService.updateWorkflowDefinitionLink(
+			user.getUserId(), group.getCompanyId(), group.getGroupId(),
+			JournalFolder.class.getName(),
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID,
+			JournalArticleConstants.DDM_STRUCTURE_ID_ALL, "Single Approver", 1);
+	}
+
 	private void _assertJournalArticleNotifications(
 			JournalArticle article, int emailNotificationCount,
-			int notificationType, int userNotificationCount)
+			int notificationType, int userNotificationCount, User user)
```

Comment:
@ak-ragnor, sorry, didn't see this before. Better use another name for user, otherwise it can be confused with the global user variable. Maybe subscribedUser?